### PR TITLE
[IMP] hr_holidays: migrate calendar views to Owl

### PR DESCRIPTION
**Pull request by kbapt**

### CODE REVIEW

File: addons/hr_holidays/views/hr_leave_allocation_views.xml [hunk 1]
```diff
@@ -63,7 +63,8 @@
         <field name="model">hr.leave.allocation</field>
         <field name="priority">32</field>
         <field name="arch" type="xml">
-            <form string="Allocation Request" js_class="allocation_leave_form">
+            <form string="Allocation Request">
+            <!-- <form string="Allocation Request" js_class="allocation_leave_form"> -->
```
Comment (tivisse) [inline addons/hr_holidays/views/hr_leave_allocation_views.xml:67]: Remaining commented code :kiss:

**Comment by tivisse** [comment]
@robodoo r+ delegate=BestHRGoorooEver

**Comment by kba-odoo** [comment]
robodoo r+

**Comment by kba-odoo** [comment]
robodoo r+

**Comment by kba-odoo** [comment]
robodoo r+

**Review by Williambraecky — APPROVED** [review]

**Pull request closed without merging**